src/shared/gpt.c

```diff
@@ -171,6 +171,29 @@ bool gpt_partition_type_is_usr_verity_sig(sd_id128_t id) {
         return gpt_partition_type_from_uuid(id).id == GPT_USR_VERITY_SIG;
 }
 
+const char* gpt_partition_type_mount_point(sd_id128_t id) {
```

Comment:
i think this should be a string table, and take the designator as input, i.e. DEFINE_STRING_TABLE_LOOKUP() and stuff